docs/google/concepts.ipynb

```diff
@@ -168,9 +153,9 @@
    "source": [
     "### Quantum Programs\n",
     "\n",
-    "In Cirq, one creates a `Circuit` in Python.  If one then wants to run it using Quantum Engine, one must then upload this Circuit (or Schedule) to the Quantum Engine.  The uploaded version of the Circuit/Schedule is called a Program.  Programs are not the Python code itself, but a representation of the Circuit in a hardware specific language.  Though you will need to make sure that your circuit uses only gates and qubits compatible with the hardware (see `cirq_google.optimized_for_sycamore()` to help with this), the `Engine` class and its corresponding `sampler` will translate the circuit to this serialized form for you.\n",
+    "In Cirq, one creates a `Circuit` in Python.  If one then wants to run it using Quantum Engine, one must then upload this Circuit to the Quantum Engine.  The uploaded version of the Circuit is called a Program.  Programs are not the Python code itself, but a representation of the Circuit in a hardware specific language (called protocol buffers).  The `Engine` class and its corresponding `sampler` will translate the circuit to this serialized form for you.  You will need to make sure that your circuit uses only gates and qubits compatible with the hardware (see `cirq_google.optimized_for_sycamore()` to help with this).  \n",
```

Comment:
I think talking about protocol buffers here is a bit confusing, since that's also not really the hardware "language" for quantum programs, just a serialization format. Also, I should note that this notebook overall swtiches between "users", "you", and "one" and it might be nice to clean that up, but up to you if you'd like to do that as part of this PR.
```suggestion
    "In Cirq, one creates a `Circuit` in Python.  If one then wants to run it using Quantum Engine, one must then upload this Circuit to the Quantum Engine API.  The uploaded version of the Circuit is called a Program.  Programs are not the Python code itself, but a representation of the Circuit suitable for running on hardware.  The `Engine` class and its corresponding `sampler` will translate the circuit into the format needed by the API for you.  You will need to make sure that your circuit uses only gates and qubits compatible with the hardware (see `cirq_google.optimized_for_sycamore()` to help with this).\n",
```
Also, `optimized_for_sycamore` is marked as deprecated; is there some other place we can point users that would be more future-proof?